app/src/main/java/com/sopt/now/data/service/AuthService.kt

```diff
@@ -0,0 +1,21 @@
+package com.sopt.now.data.service
+
+import com.sopt.now.data.dto.request.RequestLoginDto
+import com.sopt.now.data.dto.request.RequestSignUpDto
+import com.sopt.now.data.dto.response.ResponseLoginDto
+import com.sopt.now.data.dto.response.ResponseSignUpDto
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface AuthService {
+    @POST("member/join")
```

Comment:
```suggestion
    @POST("$MEMBER/$JOIN")
    
object KeyStorage {
    const val MEMBER = "member"
    const val JOIN = "join"
```

요롷게 상수화하면 더 좋을 것 같아요! 여기저기서 사용되니까 한번에 관리하기도 좋구욤

Reply:
url 상수화는 생각 못했네요! 감사합니다 👍